README.md

```diff
@@ -1 +0,0 @@
-# angular-boilerplate
```

Comment:
Add all the details in readme file. How we can setup this project locally. How we can execute the build and What are packages and standards we are using.